java/src/jmri/jmrix/openlcb/Bundle.properties

```diff
@@ -39,3 +40,4 @@ OlcbClockCustom = Custom Clock {0}
 AddTurnoutEntryToolTip = <html>Pair of 8 byte events, Thrown;Closed<br>02.01.57.00.00.54.00.72;02.01.57.00.00.54.00.73</html>
 AddSensorEntryToolTip = <html>Pair of 8 byte events, Active;Inactive<br>02.01.57.00.00.54.00.72;02.01.57.00.00.54.00.73</html>
 AddLightEntryToolTip = <html>Pair of 8 byte events, On;Off<br>02.01.57.00.00.54.00.72;02.01.57.00.00.54.00.73</html>
+AddReporterEntryToolTip = <html>06.xM.MM.LL.LL.LL.00.00<br>One 8 byte event, ends with two zero bytes, MMM is manufacturer, LLLLLL is a unique number, x is 8 or 9.</html>
```

Comment:
No need to change, but the recommendation is to always end the file with an empty line.

Reply:
No problem, fixed.